assets/src/components/v2/triptych/README.md

```diff
@@ -31,6 +37,7 @@
 
 - Set the version string in assets/src/components/v2/triptych/version.tsx to `current_year.current_month.current_day.1`.
 - In assets/webpack.config.js, change `publicPath` in the font config to have value `'fonts/'`.
+- If you've renamed / removed image assets, you might want to delete the corresponding folder in `/priv/static`. The folder accumulates assets without clearing old ones out, and these will be included in the built bundle!
```

Comment:
Yikes, I didn't know this. I wonder if this happens for other stuff like source files...

Reply:
right?!